docker/nodejs/express/app.js

```diff
@@ -4,8 +4,6 @@ var apm = require('elastic-apm-node').start({
   serviceName: process.env.EXPRESS_SERVICE_NAME,
   flushInterval: 1,
   maxQueueSize: 1,
-  secretToken: '1234',
```

Comment:
why did you remove the `secretToken` here instead of also reading it from the env var?

Reply:
Since there is nothing specific to integrating with express here, I moved to the simpler environment variable configuration for the nodejs agent.

This is in contrast with the flask and django integrations, where those settings are framework specific and expected to be ultimately configure the python agent.

I think I got all of the `secret_token` configs, eg the django one - https://github.com/elastic/apm-integration-testing/pull/220/files#diff-ea2db610473460b85666c49665c7893fR47 - if you could audit I'd appreciate it.